function_calling/notebooks/function_calling_guide.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
The SQL query example only works with SambaVerse. Could we add a SambaStudio option as well?

Reply:
new and improved db_query method, now working with llama3 8b, performance and reliability improved